modules/opensearch/opensearch.client.inc

```diff
@@ -376,7 +376,7 @@ function opensearch_execute_cache($request) {
 
   // Handle fulltext vs. dkabm caching of object.
   $type = OPENSEARCH_CACHE_TING_OBJECT;
-  if ($parms['objectFormat'] == 'docbook') {
+  if ($parms['format'] == 'docbook') {
```

Comment:
Why did you change this?

Reply:
Because in all situations 'objectFormat' key was completely missing.

Reply:
Does this change relate to any of the mentioned elements in the case?

Reply:
No, directly it is not related. Only reason why this was changed is that this incorrect/missing key was providing annoying notices on front page and in dblog.

Reply:
Ok, lets revert this for now as it does not relate to the issues this case should resolve.